api/config/cdr_config_perf.json

```diff
@@ -69,6 +69,20 @@
       "cdrDbName": "synth_r_2019q4_14",
       "hasFitbitData": true,
       "hasCopeSurveyData": true
+    },
+    {
+      "cdrVersionId": 5,
+      "isDefault": true,
```

Comment:
We can have only one "default" version per tier, so this needs to be false.

Reply:
👍🏻  I made v4 as default. removed default from v3.

Reply:
that works